### PR TITLE
Accesibility checks

### PR DESCRIPTION
**Pull request by YussufOB**
Hello, kindly help to review my code. The following activities were done:

1.  Includes `alt` content for all `img` elements.
2.  Includes `title` attribute to all inputs field.
3.  Provides `href` links for all anchor tags.
4.  Makes sure images are responsive.
5.  Re-access proper use of `hx` elements.
6. Update the README.md file.
7. Includes changes required by the reviewer.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -51,8 +51,8 @@ <h4>LET'S CONNECT</h4>
 
     <div class="row-wrapper">
       <section class="card-work">
-        <div class="min-card-images"><img src="./images/Snapshoot_Portfolio.png" alt=""></div>
-        <div class="max-card-images"><img src="./images/Snapshoot_Portfolio_1.png" alt=""></div>
+        <div class="min-card-images"><img src="./images/Snapshoot_Portfolio.png" alt="Portfolio Snapshot"></div>
+        <div class="max-card-images"><img src="./images/Snapshoot_Portfolio_1.png" alt="Portfolio Snapshot"></div>
```
Comment (keroloussamy) [inline index.html:53]: - [ ] Kindly when I resize the page I found some text and images don't disappear or get cut off on the `works` section and on the `about` section, Kindly fix it, so when you resize you can see the whole content (text, images, buttons).

Your design
image

image

image

File: index.html [hunk 2]
```diff
@@ -204,9 +204,9 @@ <h2>Contact me</h2>
         <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it</p>
       </div>
       <form class="form-section" action="https://formspree.io/f/xzbovwwb" method="post">
-          <input type="text" id="name" name="first_name" required placeholder="Your name" maxlength="30"/>
-          <input type="email" id="email" name="email" required placeholder="username@example.com" />
-          <textarea id="comments" name="comments" required placeholder="Write your message here" maxlength="500"></textarea>
+          <input type="text" title="First name" id="name" name="first_name" required placeholder="Your name" maxlength="30"/>
+          <input type="Email" title="email" id="email" name="email" required placeholder="username@example.com" />
+          <textarea title="Comments" id="comments" name="comments" required placeholder="Write your message here" maxlength="500"></textarea>
           <button type="submit" class="action-buttons">Get in touch</button>
```
Comment (keroloussamy) [inline index.html:208]: - [ ] Kindly this button doesn't pass with the hover case, kindly make the color darker.

image

File: index.html [hunk 3]
```diff
@@ -97,15 +97,15 @@ <h2>Multi-Post Stories</h2>
           </ul>
           <button type="button" class="action-buttons">See Project</button>
```
Comment (keroloussamy) [inline index.html:96]: - [ ] Kindly this button doesn't pass with the hover case, kindly make the color darker to pass.

image

**Review by keroloussamy — CHANGES_REQUESTED** [review]
Hi @YussufOB, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To highlight 🎯

- No linter errors.✔️
- Good code design and implementation. ✔️

## Required Changes ♻️
_Check the comments under the review._



## Optional suggestions
_Check the comments under the review._

- [ ] x

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by prolajumokeoni** [comment]
> Hello, kindly help to review my code. The following activities were done:
> 
> 1. Includes `alt` content for all `img` elements.
> 2. Includes `title` attribute to all inputs field.
> 3. Provides `href` links for all anchor tags.
> 4. Makes sure images are responsive.
> 5. Re-access proper use of `hx` elements.
> 6. Update the README.md file.
> 7. Includes changes required by the reviewer.

### OPTIONAL

- [ ] Kindly prepare a report listing all points that you have checked, and the result of your evaluation
 Your report should include:
List all points that you have checked, i.e:
- Page titles
- Image text alternatives** (alt text)
- Text headings
- Color contrast
- Resize
- Interaction
- Moving content
- Multimedia
- The basic structure of the page

#### Each point on the list needs to include the result of your evaluation:
If everything was ok, make a statement about that (e.g. "Page titles - no issues found").



- [ ] Kindly add your report   to the description of your pull request, please do this because this is part of the [project requirements 
](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/6_accessability.md)

**Review by prolajumokeoni — APPROVED** [review]
Hi @Team,
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
_Check comment under review_

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by OLIPLICHE — APPROVED** [review]
Hi @YussufOB, 

Well done in working on Optional suggestions 👏👏 , your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**